lib/reporters/base.js

```diff
@@ -189,9 +191,15 @@ var generateDiff = (exports.generateDiff = function(actual, expected) {
   } catch (err) {
     var msg =
       '\n      ' +
-      color('diff added', '+ expected') +
+      color(
```

Comment:
This change in the `catch` block does not make sense. We don't have any diff here, neither a normal (unifiedDiff) one nor an inlineDiff.

Reply:
Well, I disagree. This is similar to the regular case: it uses the current color style (inline vs normal) to display the error, just like in the normal case it uses that style to explain the color above the diff with "actual - expected". So you have to give me a reason why in this case we should always use the non-inline style to display the error.

Reply:
Yes, I see.
In either case the message is `+ expected - actual:  failed to generate Mocha diff`, and we don't have any diff. So I just prefer to keep it simple, there is no need to add any complexity.

Reply:
So what does that mean, exactly? Do you want me to fix the existing code as well then by basically removing the whole thing and just returning "failed to generate Mocha diff" without any color?

Reply:
No, just to revert your changes to the `catch` block.